inst/templates/ad_advs.R

```diff
@@ -0,0 +1,377 @@
+# Name: ADVS
+#
+# Label: Vital Signs Analysis Dataset
+#
+# Input: adsl, vs
+#        WHO_bmi_for_age_boys, WHO_bmi_for_age_girls, cdc_bmiage,
+#        who_lgth_ht_for_age_boys, who_lgth_ht_for_age_girls, cdc_htage,
+#        who_wt_for_age_boys, who_wt_for_age_girls, cdc_wtage,
+#        who_hc_for_age_boys, who_hc_for_age_girls,
+#        who_wt_for_ht_boys, who_wt_for_ht_girls, who_wt_for_lgth_boys, who_wt_for_lgth_girls
+
+library(admiral)
+library(admiraldev)
```

Comment:
admiraldev currently needed for `derive_params_growth_age()`
![image](https://github.com/pharmaverse/admiralpeds/assets/157114584/adff5307-e1a0-41b8-8805-85739e668150)

Reply:
did you run `devtools::load_all()` first? i dont think its needed because we import the required functions from {admiraldev} as part of the package so you don't need the users to install and load all of admiraldev which is very heavy. this was the same comment i had for {rlang} as i don't think you need to include either here. neither of these are included in any of the admiral core templates for example.

Reply:
Not today, I forgot. Done now, and I'll think about in the future. Thank you for the reminder.